example/tslint.json

```diff
@@ -1,36 +1,49 @@
 {
-  "rulesDirectory": [
-    "node_modules/codelyzer"
-  ],
+  "extends": "tslint:recommended",
```

Comment:
lol, Angular still uses tslint? It's long been deprecated, we should use eslint instead, but if the default template comes with it, I'm ok